app/routes/goal_routes.py

```diff
@@ -0,0 +1,85 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from app.routes.routes_helpers import validate_model
+import datetime
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+
+
+#POST endpoint CREATES A NEW Goal
+@goals_bp.route("", methods=["POST"])
+def create_Goal():
+        request_body = request.get_json()
+        try:
+            new_goal = Goal(title=request_body["title"])
+        except KeyError:
+            return {
+                "details": "Invalid data"
+            }, 400
+        db.session.add(new_goal)
+        db.session.commit()
+
+        return {f"goal": new_goal.to_dict()}, 201
+
+#GET ALL goals endpoint
+@goals_bp.route("", methods=["GET"])
+def read_all_goals():
+    goals = Goal.query.all()
+
+    goal_post = [goal.to_dict() for goal in goals]
+    return jsonify(goal_post), 200
```

Comment:
Let's organize this a bit differently:
```suggestion
    goals = Goal.query.all()
    goal_post = [goal.to_dict() for goal in goals]
    
    return jsonify(goal_post), 200
    
```
We can group the assignment variables together, then add a blank line between that and the return statement, as well as a line after the function itself to separate it form the next route